src/planscape/plan/views.py

```diff
@@ -504,6 +504,22 @@ def create_scenario(request: HttpRequest) -> HttpResponse:
         return HttpResponse(str(scenario.pk))
     except Exception as e:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
+    
+def get_scenario(request: HttpRequest) -> HttpResponse:
+    try:
+        assert isinstance(request.GET['id'], str)
```

Comment:
Could the logic here be encapsulated in something complementary to get_plan_by_id? 

"scenario_exists" might also be a misleading name - reading it, I expect to see a boolean true if a scenario is available and false otherwise.

Reply:
the logic is used to retrieve a scenario id rather than a plan id. i can take a look to see if we can consolidate this logic between functions in a follow up

renamed to 'scenario'.